examples/advanced/nlp-ner/utils/bert_ner_test_only.py

```diff
@@ -0,0 +1,178 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
Can this be moved to our unit tests? CC @YuanTingHsieh

Reply:
What is the purpose of this file?
From the code it does not look like a unit test though.

Reply:
it is used for testing the performance of the resulting global model using the test set, I changed the name a bit, we can further change it to avoid confusion